src/v/finjector/hbadger.h

```diff
@@ -65,6 +65,16 @@ class honey_badger {
     void register_probe(std::string_view, probe* p);
     void deregister_probe(std::string_view);
 
+    bool is_enabled() const {
```

Comment:
mark final?

Reply:
can not mark final since we do not override method